frontend/src/pages/LoginPage.jsx

```
@@ -5,125 +1,178 @@
+                  <path
+                    strokeLinecap="round"
+                    strokeLinejoin="round"
+                    d="M15 12a3 3 0 1 1-6 0 3 3 0 0 1 6 0Z"
+                  />
+                </svg>
+              )}
+            </div>
+          </div>
+
+          <div className={styles.errorZone}>
+            <p onClick={() => navigate("/recover")}>Forgot password?</p>
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)